src/Administration/Resources/app/administration/src/module/sw-product/view/sw-product-detail-specifications/sw-product-detail-specifications.html.twig

```diff
@@ -44,22 +44,26 @@
     {% endblock %}
 
     {% block sw_product_detail_specifications_custom_fields %}
-    <mt-card
+    <div
```

Comment:
this affects all the `mt-card`s here and in the other file I posted in the Slack thread. 

I tested it with computed property and used e.g. on the `essential_characteristics`. it also does not work correctly :grimacing: 

It should be fixed for all

Reply:
Okay, thought it was just the module. Good hint. Then I will look for a general solution in the card so that the v-show works again

Reply:
Should work now globally 👍

Reply:
nice :ok_hand: